src/element-stub.js

```diff
@@ -27,6 +27,11 @@ export class ElementStub extends BaseElement {
   }
 
   /** @override */
+  getPriority() {
+    throw new Error('Cannot get priority of stubbed element');
```

Comment:
@cramforce not quite `dev.assert` is DCE'd so it won't be in prod errors reporting. But, I agree, let's leave `new Error()` here so that we actually do get the error.